tf/examples/EMI-RNN/00_emi_lstm_example.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
We do prefer retaining the outputs from training runs to give users a reference output (and reference numbers). Could you please run these notebooks afresh and push.

In the same vein, it would be nice if you push a clean run --- without cells being run twice or warnings that this notebook has already been run once before etc.